### PR TITLE
integrate d3 visualization to grafana panel

### LINKED ISSUE #1262: vis: integrate `Global` vis into a grafana panel
remove the files added in #1261, moving the visualization into a new grafana panel in `ott-vis`

cc @cjrkoa

### PR DESCRIPTION
**Pull request by cjrkoa**
Fixes #1262

### CODE REVIEW

File: packages/ott-vis/src/components/SimplePanel.tsx [hunk 1]
```diff
@@ -48,13 +49,13 @@ export const SimplePanel: React.FC<Props> = ({ options, data, width, height }) =
 				viewBox={`-${width / 2} -${height / 2} ${width} ${height}`}
 			>
 				<g>
-					<circle style={{ fill: theme.colors.primary.main }} r={100} />
+					<ForceGraph />
 				</g>
```
Comment (dyc3) [inline packages/ott-vis/src/components/SimplePanel.tsx:53]: `ForceGraph` renders an svg, so putting it here makes it an svg inside another svg. Is this intentional?

**Comment by dyc3** [comment]
> Docker isn't active

Did you make sure to start it _outside_ your dev container?

File: packages/ott-vis/src/components/SimplePanel.tsx (hunk 1 above)
Comment (cjrkoa) [reply packages/ott-vis/src/components/SimplePanel.tsx:53]: Not intentional

**Comment by dyc3** [comment]
Also go ahead and remove the `d3-visualization` folder with the original code you wrote in it

**Review by dyc3 — APPROVED** [review]

**Pull request merged**